### DISCUSSION TITLE
No further information on failed deploy

### DISCUSSION THREAD

**Discussion by wutangpaul** [vercel/vercel#5837] [Help]
A push failed for me last night and this is the only information I have about it. There are no further details to be found as to what happened, and there is nothing in the build logs on my Vercel dashboard. 

<img width="355" alt="Screen Shot 2021-02-12 at 9 58 24 AM" src="https://user-images.githubusercontent.com/31920/107756128-ab467c80-6d1b-11eb-91de-3de19bf2c064.png">

**Comment by mcsdevv** [comment]
Hi there, have you tried looking at the individual commit view?
